HackYourFuture-CPH/rate-my-cv
# PageHeader to Upload CV, issue_41 done

# Description

A search page header is created where you can upload your CV,.,(Nb, currently the button is STATIC, it will fetch CVs later in the project)

Fixes # 41

# How to test?
1. run npm run story and see the component come up as below.
<img width="943" alt="Header4UploadingCV_storybook" src="https://user-images.githubusercontent.com/58881390/106162688-ad8bd100-6188-11eb-9930-570f52aac72a.PNG">

2. You can also test it in localhost npm run dev but u must import PageHeader component in app.js (Client-side), and the component will be like below; both at full screen and at mobile platform;;;
<img width="953" alt="Header4UploadingCV" src="https://user-images.githubusercontent.com/58881390/106162953-efb51280-6188-11eb-8ab3-71de2d457f64.PNG">
<img width="178" alt="Header4UploadingCV_mobile" src="https://user-images.githubusercontent.com/58881390/106162974-f5aaf380-6188-11eb-98e4-94a106274cf6.PNG">




# Checklist

- [*] I have performed a self-review of my own code
- [*] I have commented my code, particularly in hard-to-understand areas
- [*] I have made corresponding changes to the documentation
- [*] This PR is ready to be merged and not breaking any other functionality
